dbms/src/Client/Benchmark.cpp

```diff
@@ -346,13 +361,15 @@ int main(int argc, char ** argv)
 		desc.add_options()
 			("help", "produce help message")
 			("concurrency,c", boost::program_options::value<unsigned>()->default_value(1), "number of parallel queries")
-			("delay,d", boost::program_options::value<double>()->default_value(1), "delay between reports in seconds")
+			("delay,d", boost::program_options::value<double>()->default_value(1), "delay between reports in seconds (set 0 to disable)")
 			("host,h", boost::program_options::value<std::string>()->default_value("localhost"), "")
 			("port", boost::program_options::value<UInt16>()->default_value(9000), "")
 			("user", boost::program_options::value<std::string>()->default_value("default"), "")
 			("password", boost::program_options::value<std::string>()->default_value(""), "")
 			("database", boost::program_options::value<std::string>()->default_value("default"), "")
 			("stage", boost::program_options::value<std::string>()->default_value("complete"), "request query processing up to specified stage")
+			("loops,l", boost::program_options::value<size_t>()->default_value(0), "number of tests repetions")
+			("timelimit,t", boost::program_options::value<double>()->default_value(0.), "stop repeating after specified time limit")
```

Comment:
I deleted Benchmark.cpp changes from this PR.

There are some additional problems with current version of the benchmark.
I will resolve them and make separated PR.